sunpy/map/mapbase.py

```diff
@@ -174,6 +174,7 @@ def __init_subclass__(cls, **kwargs):
         `is_datasource_for` attribute.
         This is then passed into the Map Factory so we can register them.
         """
+        cls.__doc__ = cls.__doc__ + '\n' + GenericMap.__doc__.split("data array")[1].split("Examples")[0]
```

Comment:
Not very pretty but it cuts out only the parameters section. Leaves the notes to the side.